ods_ci/tests/Tests/500__jupyterhub/test-pipChanges-not-permanent.robot

```diff
@@ -32,4 +32,4 @@ Test Setup
     Begin Web Test
     Wait for RHODS Dashboard to Load
     Launch JupyterHub Spawner From Dashboard
-    Spawn Notebook With Arguments
+    Spawn Notebook With Arguments    image=s2i-minimal-notebook
```

Comment:
Can you change it for `image=minimal-notebook`?  This will make it compatible with ODH too

Reply:
updated